ksqldb-functional-tests/src/test/resources/query-validation-tests/create-struct.json

```diff
@@ -86,6 +86,19 @@
         "message": "Duplicate field names found in STRUCT"
       }
     },
+    {
+      "name": "duplicate structs in array",
```

Comment:
I forgot to tell you offline, but whenever you add a new test to the QTT (QueryTranslationTest) suite you need to run `PlannedTestGeneratorTest` manually (and remove the `@Ignore` annotation) to generate historical plans. These historical plans don't change across versions so we make sure that future versions running the old code will not deviate in behavior.

Reply:
Oops I forgot to uncomment the @ignore in the PlannedTestGeneratorTest, will submit new change shortly!